packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -141,35 +149,70 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 	return (
 		<Stack className="container-summary-view">
```

Comment:
It would probably be a good idea to break this tree down into sub-components to isolate aspects. E.g. a component for the overall table, one for the header, one for the body. We can go further if needed, but that might be a good starting place to isolate aspects and make the code a bit easier to reason about.

Reply:
(Unless FluentUI offers a more streamlined way to craft the table tree)

Reply:
+1
Typically, the frameworks give you the building blocks to make a table and its up to you to organize it.
As you said, what I've see is it's typically broken in a series of higher level components e.g:

/table
 /table-row
 /table-footer
 /table-header